server/controllers/entities/lib/get_entity_by_uri.ts

```diff
@@ -0,0 +1,15 @@
+import { getEntitiesByUris } from './get_entities_by_uris.js'
+
+interface GetEntityByUriArgs {
+  uri: string
```

Comment:
it looks like there is an attempt in 044fc55 to change file extension `.js`->`.ts`
despite PR comment
> I would have prefered to rename all files to .ts ...

The new ts file `get_entities_by_uris.ts` has not been typed (ie. `response.entities = keyBy(response.entities, 'uri')`), thus ending up crashing the build.

Shall `get_entities_by_uris` be renamed with `.js` extension ?